cities/luebeck.json

```diff
@@ -0,0 +1,162 @@
+                    53.86673
+                ]
+            }
+        },
+        {
+            "properties" : {
+                "title" : "Wochenmarkt am Brink",
+                "location" : "Am Brink",
+                "opening_hours" : "Di, Do, Sa 08:00-13:00"
+            },
+            "type" : "Feature",
+            "geometry" : {
+                "type" : "Point",
+                "coordinates" : [
+                    10.6951357,
+                    53.8576292
+                ]
+            }
+        },
+        {
+            "properties" : {
+                "title" : "Wochenmarkt am Broilingplatz",
+                "location" : "Brolingstraße",
+                "opening_hours" : "Mi 08:00-13:00"
+            },
+            "type" : "Feature",
+            "geometry" : {
+                "type" : "Point",
+                "coordinates" : [
+                    10.67126,
+                    53.87587
+                ]
+            }
+        },
+        {
+            "properties" : {
+                "title" : "Wochenmarkt am Hasenweg",
+                "location" : "Hasenweg",
+                "opening_hours" : "Mi 08:00-13:00"
+            },
+            "type" : "Feature",
+            "geometry" : {
+                "type" : "Point",
+                "coordinates" : [
+                    10.69897,
+                    53.87693
+                ]
+            }
+        },
+        {
+            "properties" : {
+                "title" : "Wochenmarkt Schlutup",
+                "location" : "Am Schlutuper Markt/Mecklenburger Straße",
+                "opening_hours" : "Mi 08:00-13:00"
+            },
+            "type" : "Feature",
+            "geometry" : {
+                "type" : "Point",
+                "coordinates" : [
+                    10.79967,
+                    53.88753
+                ]
+            }
+        },
+        {
+            "properties" : {
+                "title" : "Wochenmarkt Moislingen",
+                "location" : "Hanseplatz",
+                "opening_hours" : "Fr 08:00-13:00"
+            },
+            "type" : "Feature",
+            "geometry" : {
+                "type" : "Point",
+                "coordinates" : [
+                    10.66256,
+                    53.85937
+                ]
+            }
+        },
+        {
+            "properties" : {
+                "title" : "Wochenmarkt Kücknitz",
+                "location" : "Kirchplatz",
+                "opening_hours" : "Fr 08:00-13:00"
+            },
+            "type" : "Feature",
+            "geometry" : {
+                "type" : "Point",
+                "coordinates" : [
+                    10.805202,
+                    53.912991
+                ]
+            }
+        }
+
+    ],
+    "metadata" : {
+        "data_source" : {
+            "url" : "https://www.luebeck-tourismus.de/einkaufen/shoppingparadies/maerkte-regionale-produkte.html",
+            "title" : "Lübeck Tourismus"
```

Comment:
Please consider adding a time stamp so users know when the data has been updated.
Example: [berlin.json](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/cities/berlin.json?short_path=7185eea#L2390)

Reply:
Timestamp is added and errors should also be fixed.

Reply:
:+1: Nice. Thanks.